### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by Jmikeydarby**
#62

### CODE REVIEW

File: public/stylesheets/main.css [hunk 1]
```diff
@@ -1,5 +1,29 @@
 body {
-  background: #ffffff; }
+  background: #DDE1E6; }
 
 .align-right {
   text-align: right; }
+
+.navbar {
```
Comment (bethqiang) [inline public/stylesheets/main.css]: I'd almost like to see all of the navbar related styles in a different file, as the intention for this one was just for classes that are used site-wide and not necessarily for one component. I'm not picky one way or another though, unless this file starts getting a lot bigger...

File: app/components/Navbar.jsx [hunk 2]
```diff
@@ -6,13 +6,55 @@ import Login from './Login';
...
+          </ul>
+          <form className="navbar-form navbar-left">
+            <div className="form-group">
+              <input type="text" className="form-control" placeholder="Search"/>
+            </div>
+            <button type="submit" className="btn btn-default">Submit</button>
+          </form>
+          <ul className="nav navbar-nav navbar-right">
+            { /* Login and Logout are here temporarily, they will be moved to the Account dropdown menu eventually */ }
+            <li className="nav-user">{props.user ? null : <Signup/>}{props.user === null || Object.keys(props.user).length === 0 ? <Login/> : <WhoAmI/>}</li>
```
Comment (bethqiang) [inline app/components/Navbar.jsx]: Thanks for fixing this! :)

File: public/stylesheets/main.css (hunk 1 above)
Comment (Jmikeydarby) [inline public/stylesheets/main.css]: I thought about this, but remembered that the navbar would in theory be used site wide.  I can very easily move them to their own css file if you two want.

Comment (bethqiang) [inline public/stylesheets/main.css]: By "site-wide" I just meant like classes that would be used in multiple components, not necessarily components that would be used across the site. :)

**Pull request merged**